src/deepsparse/yolo/pipelines.py

```diff
@@ -158,7 +191,7 @@ def process_inputs(self, inputs: YOLOInput) -> List[numpy.ndarray]:
 
             if isinstance(image, str):
                 image = cv2.imread(image)
-                image = cv2.resize(image, dsize=self.input_shape)
+                image = cv2.resize(image, dsize=tuple(reversed(self.image_size)))
```

Comment:
is this cause channel dim is last in cv2?

Reply:
I don't think that's it, but this was required for shapes to line up